### PR TITLE
feat: [SSCA-2058]: Add ssca cicd tool icon

### PR DESCRIPTION
**Pull request by deepesh-ui**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2530

**Review by shash-harness — APPROVED** [review]

File: packages/icons/src/tools.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8.67 7.168a2.951 2.951 0 0 0 5.038-2.105c-.002-.379-.448-.502-.716-.234l-1.04 1.04a.5.5 0 0 1-.708 0L9.968 4.594a.5.5 0 0 1 0-.707l1.04-1.04c.268-.268.145-.714-.234-.717A2.951 2.951 0 0 0 8.67 7.167Z" stroke="#9293AB"/><path d="m7.89 5.964-4.229 4.229c-.935.935-1.695 1.41-.56 2.544 1.134 1.134 1.609.374 2.544-.56l4.229-4.23" stroke="#9293AB"/></svg>
```
Comment (avinashmadhwani02) [inline packages/icons/src/tools.svg:1]: Please add currentColor instead of harcoded color

Comment (deepesh-ui) [reply packages/icons/src/tools.svg]: done

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2531

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Pull request merged**